oracle12/pom.xml

```diff
@@ -74,4 +74,18 @@
             <version>12.1.0.2</version>
         </dependency>
     </dependencies>
+
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-surefire-plugin</artifactId>
+                    <configuration>
+                        <forkCount>1</forkCount>
```

Comment:
Have to spin up a whole VM for oracle builds--impractical to try spinning the database up and down for each test class.